packages/wallet/src/index.ts

```diff
@@ -61,8 +88,83 @@ export const parseERC20Deposit = (payload: Payload): ERC20Deposit => {
     return { success, token, sender, amount };
 };
 
+/**
+ * Decode input according to https://github.com/cartesi/rollups/tree/v1.2.0#input-encodings-for-deposits
+ * @param payload input payload
+ * @returns
+ */
+export const parseERC721Deposit = (payload: Payload): ERC721Deposit => {
+    const token = getAddress(slice(payload, 0, 20)); // 20 bytes for address
+    const sender = getAddress(slice(payload, 20, 40)); // 20 bytes for address
+    const commonPayload = slice(payload, 40);
```

Comment:
This parsing is not what documentation says

Reply:
Can you explain more about this? Does this refers to the `baseLayerData` and `execLayerData`?